### ISSUE TITLE
not support redmine 3.3.x

### ISSUE THREAD

**Issue by archonwang** [noma4i/Redmine-Touch#12]
I tried it on Redmine 3.3.x, and found it could not support it . I defined some condition with updated by , but it seems query nothing about the condition. or  it support Chinese as the condition? 

@noma4i , Thanks.

**Comment by noma4i** [comment]
Hey! I will check it with redmine 3.3.x. 
By the way you are finding plugin not to work as expected or while you fiddle with queries?

**Comment by archonwang** [comment]
@noma4i , yes.  when I query with last updated it not worked well because it could search nothing ,but there's some records in database actually.
